guides/common/modules/proc_migrating-to-external-databases.adoc

```diff
@@ -5,22 +5,15 @@ Back up and transfer existing data, then use the `{foreman-installer}` command t
 
 .Prerequisite
 
-* You have installed and configured a PostgreSQL server on a {RHEL} server.
+* You have installed and configured an external PostgreSQL server.
 
 .Procedure
 
-. On {ProjectServer}, stop {Project} services:
+. On {ProjectServer}, stop services:
```

Comment:
Should we be specific about the type of services we are stopping? As there can be many other active services.

Reply:
Possibly. I wasn't sure. To me it felt odd because PostgreSQL is considered a project service, but we exclude it. And you could write out `stop {Project} services except PostgreSQL` but I like short instructions when the actual command is self explanatory. However, I realize I may not be representative for the target audience so I'm fine with whatever the writers suggest is best here.

Reply:
```suggestion
. On {ProjectServer}, stop all {Project} services except for PostgreSQL:
```

Unsure about "except _for_"; maybe just "except"?

Reply:
`except for PostgreSQL`